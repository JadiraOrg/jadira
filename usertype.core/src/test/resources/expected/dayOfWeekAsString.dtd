<!ELEMENT dataset (
    DAYOFWEEKASSTRING*)>

<!ELEMENT DAYOFWEEKASSTRING EMPTY>
<!ATTLIST DAYOFWEEKASSTRING
    ID CDATA #REQUIRED
    DAYOFWEEK CDATA #IMPLIED
    NAME CDATA #IMPLIED
>

