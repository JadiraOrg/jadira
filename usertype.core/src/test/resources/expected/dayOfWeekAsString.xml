<?xml version='1.0' encoding='UTF-8'?>
<dataset>
  <dayOfWeekAsString ID="0" DAYOFWEEK="MONDAY" NAME="test_0"/>
  <dayOfWeekAsString ID="1" DAYOFWEEK="TUESDAY" NAME="test_1"/>
  <dayOfWeekAsString ID="2" DAYOFWEEK="SUNDAY" NAME="test_2"/>
</dataset>
